client/src/config/socketIO.js

```diff
@@ -0,0 +1,5 @@
+import io from 'socket.io-client';
+
+const socket = io(process.env.REACT_APP_API_URL);
```

Comment:
import env from a common file, there you can set defaults also.

Reply:
Updated.